app/main.py

```diff
@@ -1,8 +1,38 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+        self.name = name
+        self.age = age
+
+        Person.people[self.name] = self
+
+
+def find_partner(person_obj: Person,
```

Comment:
Your function name is ambiguous since the function itself does not only find a partner but also sets the `wife` or `husband` attribute to it. Either rename the function or modify its logic.